### DISCUSSION TITLE
Bonus to healing if the patient is in a hospital bed

### DISCUSSION THREAD

**Discussion by FarrunDragon** [FakeFishGames/Barotrauma#8816] [Ideas and suggestions]
I think a hospital bed could be improved by adding bonuses when doctoring patients. If a player has internal damage, being patched up while trying to walk around wouldn't be as effective as putting them in a bed and giving them a full fix. The best thing I think would be a combination of minor bonuses, including a skill buff, so it's less likely a treatment will have bad side effects, along with a bonus to the amount healed. In addition to that, beds would get a bonus to healing over time if the player stays in the bed after being healed.

**Comment by James-E-A** [comment]
Alternative suggestion: nerf [Double Duty](https://barotraumagame.com/wiki/Double_Duty) so that the +20 medical bonus only applies to patients in the medbay or a hospital bed
